queries.sql

```diff
@@ -9,3 +9,50 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg <= 17.3 AND weight_kg >= 10.4;
 
+/* transactions */
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+COMMIT;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE neme LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS null;
+COMMIT;
```

Comment:
- [ ] As previous reviewer suggested you to verify that change was made 🤔 So you can verify the change using SELECT statement 👍 You query will like below ⬇️
```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```